src/components/Form/CrossPodQueryForm.jsx

```diff
@@ -76,42 +82,42 @@ const CrossPodQueryForm = () => {
     }
   };
 
-  const formRowStyle = {
-    margin: '20px 0'
-  };
-
-  /* eslint-disable jsx-a11y/label-has-associated-control */
   return (
     <FormSection
       title="Cross Pod Search"
       state={state}
       statusType="Search status"
       defaultMessage="To be searched..."
     >
-      <form onSubmit={handleCrossPodQuery} autoComplete="off">
-        <div style={formRowStyle}>
-          <label htmlFor="cross-search-doc">Search document from username: </label>
-          <br />
-          <br />
-          <input
-            id="cross-search-doc"
-            size="25"
-            name="crossPodQuery"
-            {...username}
-            placeholder={selectedUser}
-          />
-        </div>
-        <div style={formRowStyle}>
-          <label htmlFor="cross-search-doctype">Select document type to search: </label>
-          <DocumentSelection htmlId="cross-search-doctype" />{' '}
-          <button disabled={state.processing} type="submit">
-            Search Pod
-          </button>
-        </div>
-      </form>
+      <Box display="flex" justifyContent="center" sx={{ minWidth: 120 }}>
+        <form onSubmit={handleCrossPodQuery} autoComplete="off">
+          <FormControl variant="standard" fullWidth>
+            <TextField
+              id="cross-search-doc"
+              name="crossPodQuery"
+              {...username}
+              placeholder={selectedUser}
+              label="Enter username"
+              InputProps={{
+                type: 'search'
+              }}
```

Comment:
This looks like something that would be useful during validation, but I don't think we need this just yet. 

Not until we figure out what to do with our username/webId system at the very least. Can be removed for now.